File: applicatie/src/main/java/com/vdzon/administratie/rubriceren/rest/RubriceerResource.java
package com.vdzon.administratie.rubriceren.rest;

import com.vdzon.administratie.util.JsonUtil;
import spark.Spark;

import javax.inject.Inject;


public class RubriceerResource {

    @Inject
    public RubriceerResource(RubriceerService rubriceerService) {
        Spark.get("/rest/rebriceerregels/", rubriceerService::getRubriceerRegels, JsonUtil.json());
        Spark.post("/rest/rubriceer/", rubriceerService::rubriceerRegels, JsonUtil.json());
    }


}

File: applicatie/src/main/java/com/vdzon/administratie/rest/rubriceren/RubriceerResource.kt
package com.vdzon.administratie.rest.rubriceren

import com.vdzon.administratie.util.JsonUtil
import spark.Route
import spark.Spark

import javax.inject.Inject


class RubriceerResource
@Inject
constructor(rubriceerService: RubriceerService) {
    init {
        Spark.get("/rest/rebriceerregels/", Route { req, res -> rubriceerService.getRubriceerRegels(req, res) }, JsonUtil.json())
        Spark.post("/rest/rubriceer/", Route { req, res -> rubriceerService.rubriceerRegels(req, res) }, JsonUtil.json())
    }


}
